superset-frontend/src/dashboard/components/SliceHeaderControls.jsx

```diff
@@ -171,11 +171,18 @@ class SliceHeaderControls extends React.PureComponent {
       addDangerToast,
       isFullSize,
     } = this.props;
-    const cachedWhen = moment.utc(cachedDttm).fromNow();
+    const cachedWhen = cachedDttm.map(itemCachedDttm =>
+      moment.utc(itemCachedDttm).fromNow(),
+    );
     const updatedWhen = updatedDttm ? moment.utc(updatedDttm).fromNow() : '';
-    const refreshTooltip = isCached
-      ? t('Cached %s', cachedWhen)
-      : (updatedWhen && t('Fetched %s', updatedWhen)) || '';
+    const getCachedTitle = itemCached => {
+      return itemCached
+        ? t('Cached %s', cachedWhen)
+        : updatedWhen && t('Fetched %s', updatedWhen);
+    };
+    let refreshTooltip = isCached.map(getCachedTitle) || '';
+    // If all queries have same cache time we can unit them to one
+    refreshTooltip = [...new Set(refreshTooltip)].join(', ');
```

Comment:
@villebro So basically if all requests will have same time it will look like"
![Screen Shot 2020-12-02 at 15 04 00](https://user-images.githubusercontent.com/56388545/100876256-d1e06f80-34af-11eb-8aeb-9ecd1af71c49.png)
And if different then like this:
![Screen Shot 2020-12-02 at 15 05 37](https://user-images.githubusercontent.com/56388545/100876280-d9a01400-34af-11eb-9101-a00560d32b94.png)

Reply:
I wonder if we should instead have something like this for the case when there are more than one result:
```
*Force refresh*
Query 1: Fetched a minute ago
Query 2: Fetched 3 minutes ago
Query 3: Not cached
```
Also, we should think about how to display multiple results in the "View Results" tab. But that can wait; let's just make sure the user experience is unchanged for single queries.

Reply:
1. updated

Reply:
1. done

Reply:
@simchaNielsen 
could you share the new screenshot?

Reply:
![Screen Shot 2020-12-02 at 15 51 09](https://user-images.githubusercontent.com/56388545/100970573-df443b00-353d-11eb-9b30-d69df0b3080f.png)

Reply:
![image](https://user-images.githubusercontent.com/56388545/100970630-f8e58280-353d-11eb-8499-7b827d3be9ec.png)